index.bs

```diff
@@ -4595,21 +4595,24 @@ in several ways, including:
     signature using the [=ECDAA-Issuer public key=], but the attestation signature does not serve as a global correlation handle.
 
 
-## Authentication Ceremonies ## {#sec-assertion-privacy}
+## Registration and Authentication Ceremonies Privacy ## {#sec-assertion-privacy}
 
-In order to protect the user from being identified without [=user consent|consent=], implementations of the
-{{PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)}} method need to take care to
-not leak information that could enable the [=[RP]=] to distinguish between these cases:
+In order to protect users from being identified without [=user consent|consent=], implementations of the
+{{PublicKeyCredential/[[Create]](origin, options, sameOriginWithAncestors)}} and
+{{PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)}} methods need to take care to
+not leak information that could enable a malicious [=[RP]=] to distinguish between these cases, where "named" means that the
+[=public key credential|credential=] is listed by the [=[RP]=] in either {{MakePublicKeyCredentialOptions/excludeCredentials}}
+or {{PublicKeyCredentialRequestOptions/allowCredentials}}, as applicable:
 
 - A named [=public key credential|credential=] is not available.
 - A named [=public key credential|credential=] is available, but the user does not [=user consent|consent=] to use it.
 
 If the above cases are distinguishable, information is leaked by which a malicious [=[RP]=] could identify the user by probing for
 which [=public key credential|credentials=] are available. For example, one such information leak is if the client returns a
 failure response as soon as the user denies [=user consent|consent=] to proceed with the operation. In this case the [=[RP]=]
```

Comment:
Perhaps change

>as soon as the user denies [=user consent|consent=] to proceed with the operation

to something like this?

>as soon as the user denies [=user consent|consent=] to proceed with an [=authentication=] [=ceremony=]

Seeing as the following sentence refers specifically to `allowCredentials`, I mean.

Reply:
incorp'd a build on this suggestion, pls review

Reply:
What I was trying to say is that the conclusion "at least one of the credentials listed in the allowCredentials parameter is available" is applicable only to the case of authentication ceremonies, not registration ceremonies. For registration ceremonies the conclusion is instead that the user has an authenticator that is not registered, which probably isn't as big a privacy concern.